frontend/src/react-app-env.d.ts

```diff
@@ -1 +1 @@
-/// <reference types="react-scripts" />
+// / <reference types="react-scripts" />
```

Comment:
I don't think this file should be on GitHub. Add it to the `.gitignore` file that is in the `frontend` folder. This will make sure that GitHub does not keep track of this file.

Also in the `.gitignore` that is in the base directory, add `.DS_Store` and `server/.env`. Thanks!